src/discussions/post-comments/comments/comment/CommentEditor.jsx

```diff
@@ -151,7 +193,10 @@ const CommentEditor = ({
             id={editorId}
             value={values.comment}
             onEditorChange={formikCompatibleHandler(handleChange, 'comment')}
-            onBlur={formikCompatibleHandler(handleBlur, 'comment')}
+            onBlur={(content) => {
+              formikCompatibleHandler(handleChange, 'comment');
+              saveDraftContent(typeof content === 'object' ? content.target.getContent() : content);
```

Comment:
refactor it
saveDraftContent(typeof content === 'object' ? content.target.getContent() : content);
`saveDraftContent(content)`